official/utils/testing/reference_data/ResNet/batch-size-32_building_version-1_width-8_channels-4/results.json

```diff
@@ -0,0 +1 @@
+[32, 8, 8, 4, 0.23128163814544678, 0.22117376327514648, 4100.51806640625, 32, 8, 8, 4, 0.9646798372268677, 0.16614516079425812, 5799.6708984375]
```

Comment:
Why are the names camel cased? Seems preferable to just lowercase everything, rather than try to remember how to capitalize all these model names. I will point out that cifar10 is actually CIFAR-10, so, there's no hope if you try to stick with how things pretend to be spelled.

Reply:
Hard to argue with nihilism. Lower case it is.